src/ui/trackData.ts

```diff
@@ -0,0 +1,478 @@
+import genes_interpretation_2 from '../script/img/popover-images/genes/interpretation_2.png';
+import genes_interactions_1 from '../script/img/popover-images/genes/interactions_1.png';
+import genes_interactions_2 from '../script/img/popover-images/genes/interactions_2.png';
+import mutations_interpretation_1 from '../script/img/popover-images/mutations/interpretation_1.png';
+import mutations_interactions_1 from '../script/img/popover-images/mutations/interactions_1.png';
+import mutations_interactions_2 from '../script/img/popover-images/mutations/interactions_2.png';
+import indel_interpretation_1 from '../script/img/popover-images/indel/interpretation_1.png';
+import indel_interpretation_2 from '../script/img/popover-images/indel/interpretation_2.png';
+import indel_interactions_1 from '../script/img/popover-images/indel/interactions_1.png';
+import copy_number_variants_interpretation_1 from '../script/img/popover-images/copy_number_variants/interpretation_1.png';
+import copy_number_variants_interpretation_2 from '../script/img/popover-images/copy_number_variants/interpretation_2.png';
+import copy_number_variants_interactions_1 from '../script/img/popover-images/copy_number_variants/interactions_1.png';
+import gains_interpretation_1 from '../script/img/popover-images/gains/interpretation_1.png';
+import gains_interpretation_2 from '../script/img/popover-images/gains/interpretation_2.png';
+import loh_interpretation_1 from '../script/img/popover-images/loh/interpretation_1.png';
+import loh_interpretation_2 from '../script/img/popover-images/loh/interpretation_2.png';
+import structural_variants_interpretation_1 from '../script/img/popover-images/structural_variants/interpretation_1.png';
+import structural_variants_interactions_1 from '../script/img/popover-images/structural_variants/interactions_1.png';
+import structural_variants_interactions_2 from '../script/img/popover-images/structural_variants/interactions_2.png';
+import coverage_interpretation_1 from '../script/img/popover-images/coverage/interpretation_1.png';
+import coverage_interpretation_2 from '../script/img/popover-images/coverage/interpretation_2.png';
+import alignment_interpretation_1 from '../script/img/popover-images/alignment/interpretation_1.png';
+import alignment_interpretation_2 from '../script/img/popover-images/alignment/interpretation_2.png';
+import alignment_interpretation_3 from '../script/img/popover-images/alignment/interpretation_3.png';
+import alignment_interactions_1 from '../script/img/popover-images/alignment/interactions_1.png';
+import sequence_interpretation_1 from '../script/img/popover-images/sequence/interpretation_1.png';
+
+export type Track =
+    | 'ideogram'
+    | 'gene'
+    | 'mutation'
+    | 'cnv'
+    | 'sv'
+    | 'indel'
+    | 'driver'
+    | 'gain'
+    | 'loh'
+    | 'coverage'
+    | 'sequence'
+    | 'alignment';
+
+// TODO: Not ideal to hard coded!
+// The heights of each track
+export const getTrackData = (
```

Comment:
Now that this contains the documents, we can clarify this better with the name

```suggestion
export const getTrackDocData = (
```